src/software/ai/hl/stp/play_info.cpp

```diff
@@ -1,7 +1,7 @@
 #include "software/ai/hl/stp/play_info.h"
 
 PlayInfo::PlayInfo(std::string referee_command_name, std::string play_name,
-                   std::unordered_set<std::string> robot_tactic_assignment)
+                   std::vector<std::string> robot_tactic_assignment)
```

Comment:
Did order become important for this? Why the change?

Reply:
this is for displaying it in the visualizer in robot id order

Reply:
If the only goal here is to change how the information is displayed in the visualizer, I'd really recommend changing `std::string` to something with more information (such as a struct with a robot id and tactic reference) so that the visualizer can be responsible for deciding how things are displayed, rather then something in STP making decisions about visualization.

Reply:
added to https://github.com/UBC-Thunderbots/Software/issues/1831